components/pagination/README.md

```diff
@@ -28,10 +28,4 @@ const PaginationAnchor = asPaginationItem('a');
 ### References:
 - https://govuk-static.herokuapp.com/component-guide/previous_and_next_navigation
 
-### Properties
-Prop | Required | Default | Type | Description
-:--- | :------- | :------ | :--- | :----------
- `children` | true |  | node | `asPaginationItem` nodes
```

Comment:
is this description not valid?

Reply:
Looks to be correct from what i've observed.
```jsx
const PaginationTag = asPaginationItem('a');
const PaginationLink = asPaginationItem(Link);

<Pagination>
    <PaginationTag href="#prev" previousPage>
      Previous page
    </PaginationTag>
    <PaginationTag href="#next" nextPage>
      Next page
    </PaginationTag>
</Pagination>
```